pkg/qemu/qemu.go

```diff
@@ -812,6 +835,75 @@ func Cmdline(cfg Config) (string, []string, error) {
 	return exe, args, nil
 }
 
+func FindVirtiofsd() (string, error) {
+	type vhostUserBackend struct {
+		BackendType string `json:"type"`
+		Binary      string `json:"binary"`
+	}
+
+	const vhostConfigsDir = "/usr/share/qemu/vhost-user"
```

Comment:
Should be relative to qemu binary

https://github.com/lima-vm/lima/blob/658ef436ee7d7bf7f1c48499058fb46aed3777e4/pkg/qemu/qemu.go#L877-L926